pyrometheus/codegen/python.py

```diff
@@ -293,63 +293,62 @@ def get_mixture_internal_energy_mass(self, temperature, mass_fractions):
         return self.gas_constant * temperature * emix
 
     def get_species_viscosities(self, temperature):
-        return self._pyro_make_array([
+        return self.usr_np.sqrt(temperature)*self._pyro_make_array([
                 % for sp in range(sol.n_species):
                 ${cgm(ce.transport_polynomial_expr(sol.get_viscosity_polynomial(sp),
                                                     2, Variable("temperature")))},
                 % endfor
                 ])
 
-    def get_mixture_viscosity(self, temperature, mass_fractions):
+    def get_mixture_averaged_viscosity(self, temperature, mass_fractions):
         mole_fractions = self.iwts * mass_fractions\
  * self.get_mix_molecular_weight(mass_fractions)
-        viscosities = self.usr_np.sqrt(temperature)\
- * self.get_species_viscosities(temperature)
+        viscosities = self.get_species_viscosities(temperature)
         mix_rule_f = self._pyro_make_array([
             %for sp in range(sol.n_species):
-            ${cgm(ce.wilke_mixture_rule_expr(sol, sp, Variable("mole_fractions"),
-                                             Variable("viscosities")))},
+            ${cgm(ce.viscosity_mixture_rule_wilke_expr(sol, sp,
+                 Variable("mole_fractions"), Variable("viscosities")))},
             %endfor
             ])
         return sum(mole_fractions*viscosities/mix_rule_f)
 
     def get_species_thermal_conductivities(self, temperature):
-        return self._pyro_make_array([
+        return self.usr_np.sqrt(temperature)*self._pyro_make_array([
                 % for sp in range(sol.n_species):
                 ${cgm(ce.transport_polynomial_expr(
                     sol.get_thermal_conductivity_polynomial(sp), 1,
                     Variable("temperature")))},
                 % endfor
                 ])
 
-    def get_mixture_thermal_conductivity(self, temperature, mass_fractions):
+    def get_mixture_averaged_thermal_conductivity(self, temperature, mass_fractions):
         mole_fractions = self.iwts * mass_fractions\
  * self.get_mix_molecular_weight(mass_fractions)
-        conductivities = self.usr_np.sqrt(temperature)\
- * self.get_species_thermal_conductivities(temperature)
+        conductivities = self.get_species_thermal_conductivities(temperature)
         return 0.5*(sum(mole_fractions*conductivities)
             + 1/sum(mole_fractions/conductivities))
 
-    def get_species_diffusivities(self, temperature):
+    def get_species_mass_diffusivities(self, temperature):
```

Comment:
I think these are binary diffusivities.

Reply:
Renamed to "get_species_binary_mass_diffusivities".